kani-compiler/src/codegen_cprover_gotoc/codegen/rvalue.rs

```diff
@@ -841,6 +841,8 @@ impl<'tcx> GotocCtx<'tcx> {
             // for it. Build and insert a function that just calls an unimplemented block
             // to maintain soundness.
             let drop_sym_name = format!("{}_unimplemented", self.symbol_name(drop_instance));
+            let pretty_name =
+                format!("drop_unimplemented<{}>", self.readable_instance_name(drop_instance));
```

Comment:
Wonder if this would be a better name for the sym_name too?

Reply:
I'll change it (but leaving out the angle brackets).